src/S3SparseIterator.cpp

```diff
@@ -233,7 +233,8 @@ void S3SparseIterator::threadFunction(const Configuration& config) {
     std::cout << "Waiting for pref_sem" << std::endl;
     pref_sem.wait();
 
-    std::string obj_id_str = std::to_string(getObjId(left_id, right_id));
+    std::string obj_id_str =
+        std::string("CIRRUS") + std::to_string(getObjId(left_id, right_id));
```

Comment:
Sometimes we append "CIRRUS" to the beginning of the files in S3, othertimes we don't. We should be consistent about this.

Reply:
Not yet fixed.

Reply:
Marking as resolved, as this issue is beyond the scope of this PR.